cuqi/__init__.py

```diff
@@ -5,6 +5,7 @@
 from . import geometry
 from . import implicitprior
 from . import likelihood
+from . import mcmc
```

Comment:
I suggest we make the experimental nature more apparent by putting this mcmc module within an experimental module, ie, cuqi.experimental.mcmc